front/app/containers/SearchPage/SearchPage.tsx

```diff
@@ -504,10 +502,35 @@ class SearchPage extends React.Component<SearchPageProps, SearchPageState> {
   };
 
   componentDidMount() {
-    if (this.state.showCards) {
-      window.addEventListener('scroll', this.handleScroll);
+    let searchTerm = this.props.location.search;
+    searchTerm = searchTerm.slice(2);
+    searchTerm = searchTerm.replace(/\+/g, ' ');
+    if (this.props.location.search) {
+      let q = { key: 'AND', children: [{ children: [], key: searchTerm }] };
+      this.setState(
+        {
+          params: {
+            q: q,
+            aggFilters: [],
+            crowdAggFilters: [],
+            sorts: [],
+            page: 0,
+            pageSize: 25,
+          },
+        },
+        () => this.updateSearchParams(this.state.params)
+      );
+      if (this.state.showCards) {
+        window.addEventListener('scroll', this.handleScroll);
+      } else {
+        window.removeEventListener('scroll', this.handleScroll);
+      }
     } else {
-      window.removeEventListener('scroll', this.handleScroll);
+      if (this.state.showCards) {
```

Comment:
Can we just put these 5 lines outside of the if on line 508 instead of duplicating them?

Reply:
Yeah that works for sure. I thought that was what was causing the bug I posted about in our #code_the_dream channel. After realizing it wasn't I forgot to revert, appreciate you pointing it out